### PR TITLE
Pr update clientarch

### PR DESCRIPTION
**Pull request by xresende**

### CODE REVIEW

File: siriuspy/siriuspy/clientarch/pvarch.py [hunk 1]
```diff
@@ -3,29 +3,38 @@
...
         self.avg_bytes_per_event = None
         self.estimated_storage_rate_kb_hour = None
         self.estimated_storage_rate_mb_day = None
         self.estimated_storage_rate_gb_year = None
 
-        self.connector = client_archiver
+    @property
+    def request_url(self):
+        """."""
+        url = self.connector.getPVDetails(self.pvname, get_request_url=True)
```
Comment (fernandohds564) [inline siriuspy/siriuspy/clientarch/pvarch.py]: there is a `self.connect()` missing here.

File: siriuspy/siriuspy/clientarch/pvarch.py [hunk 2]
```diff
@@ -52,22 +63,127 @@ def update(self):
...
+        self.timestamp_stop = None
+        self.timestamp = None
+        self.value = None
+        self.status = None
+        self.severity = None
+
+    @property
+    def request_url(self):
+        """."""
+        url = self.connector.getData(
```
Comment (fernandohds564) [inline siriuspy/siriuspy/clientarch/pvarch.py]: there is a `self.connect()` missing here.

File: siriuspy/siriuspy/clientarch/pvarch.py (hunk 1 above)
Comment (xresende) [reply siriuspy/siriuspy/clientarch/pvarch.py]: done

File: siriuspy/siriuspy/clientarch/pvarch.py (hunk 2 above)
Comment (xresende) [reply siriuspy/siriuspy/clientarch/pvarch.py]: done

**Pull request merged**